ci/do_ci.sh

```diff
@@ -85,21 +94,22 @@ if [[ "$1" == "bazel.release" ]]; then
   setup_clang_toolchain
   echo "bazel release build with tests..."
   bazel_binary_build release
+  BAZEL_RELEASE_OPTIONS="test ${BAZEL_TEST_OPTIONS} -c opt ${MEMORY_TEST_EXACT_ARGS}"
```

Comment:
nit: it seems like this should be BAZEL_TEST_RELEASE_OPTIONS or something like that?

Reply:
done.